src/libgrate/grate-texture.c

```diff
@@ -274,6 +274,18 @@ void grate_texture_clear(struct grate *grate, struct grate_texture *tex,
 		grate_error("host1x_gr2d_clear() failed: %d\n", err);
 }
 
+void grate_texture_clear2(struct grate *grate, struct grate_texture *tex,
+			 uint32_t color, unsigned x, unsigned y,
+			 unsigned width, unsigned height)
```

Comment:
@digetx: Yeah, probably. But that feels like a separate change... Do as you like ;)